app/components/input-field.hbs

```diff
@@ -0,0 +1,18 @@
+<div class="field">
+  <label for={{@for}}>{{@label}}:</label>
+  <Input
```

Comment:
We should be using `<input>`, not the older `<Input>`, ever since Ember Octane edition.

https://discuss.emberjs.com/t/ember-octane-and-2-way-binding-what-are-you-recommend/18092

Reply:
Done